modules/nw-egress-ips-about.adoc

```diff
@@ -206,7 +206,7 @@ When using the manual assignment approach for egress IP addresses the following
 - You set the `egressIPs` parameter of each node's `HostSubnet` resource to indicate the IP addresses that can be hosted by a node.
 - Multiple egress IP addresses per namespace are supported.
 
-When a namespace has multiple egress IP addresses, if the node hosting the first egress IP address is unreachable, {product-title} will automatically switch to using the next available egress IP address until the first egress IP address is reachable again.
+When a namespace has multiple egress IP addresses and those addresses are hosted on multiple nodes, pods use all egress addresses roughly equally. If the node hosting one of the egress IP addresses fails then pods continue using the remaining functioning node and egress IP address after a short delay.
```

Comment:
this is mostly right, there are two scenarios, 
1. if a pod is on a node that is hosting an egressIP that pod will always use the egressIP of the node it is on
2. if a pod is not on a node hosting an egressIP it is random which which egressIP it will use